### ISSUE TITLE
All submissions are stuck at "Submitted" (for previously working competitions)

### ISSUE THREAD

**Issue by chryssa-zrv** [codalab/codabench#1546]
Three previously running fine competitions (in secret mode) cannot process submissions today as they are stuck in "submission". I only tried changing 1 reference file for this competition:

[https://www.codabench.org/competitions/3518/?secret_key=88422dfc-4e8c-4627-be47-261acaff49ec](https://www.codabench.org/competitions/3518/?secret_key=88422dfc-4e8c-4627-be47-261acaff49ec )

While the other 2 that are linked below remained unchanged.

Task 1:
[https://www.codabench.org/competitions/3520/?secret_key=62c389d2-611f-418a-92d9-09f87f1bd900](https://www.codabench.org/competitions/3520/?secret_key=62c389d2-611f-418a-92d9-09f87f1bd900)

Task3:
[https://www.codabench.org/competitions/3523/?secret_key=49f3301d-a0db-4163-9b5c-7fd24574b659](https://www.codabench.org/competitions/3523/?secret_key=49f3301d-a0db-4163-9b5c-7fd24574b659)

I uploaded the new reference zip files in Datasets and Programs and then edited the competition in Tasks. I tried reverting to the previous zip but the issue remains for all competitions

**Comment by chryssa-zrv** [comment]
@ObadaS I see the problem fixed only for Task 2 (the 1st link in my previous message) but not the other 2.

The following 2 still get stuck: 

Task 1:
[https://www.codabench.org/competitions/3520/?secret_key=62c389d2-611f-418a-92d9-09f87f1bd900](https://www.codabench.org/competitions/3520/?secret_key=62c389d2-611f-418a-92d9-09f87f1bd900)

Task3:
[https://www.codabench.org/competitions/3523/?secret_key=49f3301d-a0db-4163-9b5c-7fd24574b659](https://www.codabench.org/competitions/3523/?secret_key=49f3301d-a0db-4163-9b5c-7fd24574b659)

**Comment by Didayolo** [comment]
I close as the problem seems to be solved. Please re-open the issue if needed.

**Comment by Hypatiaalegra** [comment]
Hi, @ObadaS @Didayolo , I have also encountered the issuse right now, the benchmark I'm hosting is https://www.codabench.org/competitions/4140/ and all submissions get stuck at submitted status similarly. When I uploaded another new private benchmark https://www.codabench.org/competitions/4304/?secret_key=6ce20783-6115-48f8-b513-969629e66ea8, this issue happened again no matter what I submitted. Thanks for you help!
